### PR TITLE
Feature/red 545 mem check

### PR DESCRIPTION
**Pull request by red-tt-bear**

### CODE REVIEW

File: src/configurations.ts [hunk 1]
```diff
@@ -122,6 +122,14 @@ const configurations = {
         configSalt: readStringValue('CONFIG_SALT', ''),
         autoDeleteTemp: readBooleanValue('AUTO_DELETE_TEMP_FILES', true),
     },
+    monitoring: {
+        memory: {
```
Comment (PanopticaRising) [inline src/configurations.ts:126]: Is there a way to disable memory monitoring?

File: src/daemons.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import logger from './utilities/logger';
+import { getHeapStatistics } from 'v8';
+import configurations from './configurations';
+const {
+    debugThreshold,
+    warningThreshold,
+    errorThreshold,
+    interval: memoryDaemonInterval
+} = configurations.monitoring.memory;
+const minThreshold = Math.min(debugThreshold, warningThreshold, errorThreshold);
```
Comment (PanopticaRising) [inline src/daemons.ts:10]: Is the expected way to avoid log output to set all the environment variables to 100?

Comment (red-tt-bear) [reply src/daemons.ts:10]: Yup

File: src/configurations.ts (hunk 1 above)
Comment (red-tt-bear) [reply src/configurations.ts:126]: Set all levels above 100

**Review by PanopticaRising — APPROVED** [review]

**Pull request merged**